style.css

```diff
@@ -885,10 +885,20 @@ hr {
 }
 
 .modal-close {
```

Comment:
- [ ] [OPTIONAL] You should implement the above requirements in all repositories in your pair-programming group, kindly in the future pay attention to this requirement

Reply:
@ismailco I will. Thanks, man.